circle.yml

```diff
@@ -5,6 +5,12 @@ machine:
     - docker
     - redis
 
+dependencies:
+  pre:
+    - npm i -g get-firefox
+    - get-firefox --platform linux --extract --target /home/ubuntu/send
+    - npm i -g geckodriver
```

Comment:
I bet you can add this global install to the line 10 `npm i -g get-firefox geckodriver` and install them both at the same time.

Not sure if we'd want to add these as devDependencies if we'd ever need to run them locally. 🤷‍♀️